src/resolvers/TableViewInvitationResolver.ts

```diff
@@ -71,6 +70,48 @@ const TableViewInvitationResolver = {
         )
       }
     },
+
+    async filterInvitations(_: any, args: { limit: number; offset: number; role: string; status: string }) {
+      try {
+        const limit = args.limit ?? 5;
+        const offset = args.offset ?? 0;
+        const { role, status } = args;
+
+        if (!role &&!status) throw new GraphQLError('No filter criteria provided');
```

Comment:
Could you make it possible to retrieve invitations based on either role (e.g., TTLs) or status independently, without having to specify both? Currently, you must select both a specific role and status, but it would be more flexible if we could retrieve, for example, TTL invitations without specifying a particular status.

Reply:
Hello, @Olivier-dev We already have a search bar for that. This is one, is for retrieving invitations by a specific role with a specific status. In a few words, both specified role and status must meet.